modules/client/src/main/java/org/apache/ignite/internal/client/TcpClientChannel.java

```diff
@@ -60,6 +63,9 @@
             ProtocolVersion.V3_0_0
     );
 
+    /** Minimum supported heartbeat interval. */
+    private static final long MIN_RECOMMENDED_HEARTBEAT_INTERVAL = 50;
```

Comment:
Don't you think 50 ms is a too short for real network envirnment?

Reply:
@isapego agree, what would you suggest?

Reply:
I don't know really. How about 500 or at least 200-300?

Reply:
500 sounds good, fixed.